docs/go/how-to-handle-a-query-in-a-workflow-in-go.md

```diff
@@ -15,6 +15,8 @@ The handler must be a function that returns two values:
 1. A serializable result
 2. An error
 
+Do not include Side Effects or command generation within a query handler. This could mutate the Workflow state and lead to unexpected behaviors on subsequent runs.
```

Comment:
```suggestion
Do not include any logic that causes Command generation within a Query handler, such as Side Effects or local Activities.
This will lead to unexpected behavior.
```

Just a small stylistic note - put each sentence on a newline.
When someone makes a change, the change will be limited to just that sentence.